gdraw/ggdkcdraw.c

```diff
@@ -51,7 +51,16 @@ static void _GGDKDraw_CheckAutoPaint(GGDKWindow gw) {
             _GGDKDraw_CleanupAutoPaint(gw->display);
             gw->display->dirty_window = gw;
         }
+
+        // Shut up! I know what I'm doing.
+#ifdef GGDKDRAW_GDK_3_22
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
+#endif
```

Comment:
You can simply use `G_GNUC_BEGIN_IGNORE_DEPRECATIONS`/`G_GNUC_END_IGNORE_DEPRECATIONS` from glib.

Reply:
Thanks.